runtime/src/snapshot_package.rs

```diff
@@ -115,6 +117,7 @@ impl AccountsPackage {
             hash_for_testing,
             cluster_type: bank.cluster_type(),
             snapshot_type,
+            accounts: Arc::clone(&bank.accounts()),
```

Comment:
It looks like `bank.accounts()` returns a newly-cloned `Arc<Accounts>`:

https://github.com/solana-labs/solana/blob/6962a667e5fd53769fb133540a94bbcada279e1b/runtime/src/bank.rs#L5543-L5545

Can this be simplified then?

```suggestion
            accounts: bank.accounts(),
```